.github/workflows/post_deploy_agent.yml

```
@@ -189,3 +189,21 @@ jobs:
         env:
           BUILD_PATH: ${{ github.workspace }}/build/NugetVersionDeprecator/NugetVersionDeprecator.csproj
           PUBLISH_PATH: ${{ github.workspace }}/publish
+
+  release-tags:
+    name: Create release tags
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create release tags for Lambda and K8s Init Containers
+        run: |
+          RELEASE_TITLE="New Relic .NET Agent ${AGENT_VERSION}.0"
+          RELEASE_TAG="${AGENT_VERSION}.0_dotnet"
+          RELEASE_NOTES="Automated release for [.NET Agent ${AGENT_VERSION}](https://github.com/newrelic/newrelic-dotnet-agent/releases/tag/${AGENT_VERSION})"
+          gh auth login --with-token <<< $GH_RELEASE_TOKEN
+          echo "newrelic/newrelic-lambda-layers - Releasing ${RELEASE_TITLE} with tag ${RELEASE_TAG}"
+          gh release create "${RELEASE_TAG}" --title=${RELEASE_TITLE} --repo=newrelic/newrelic-lambda-layers --notes=${RELEASE_NOTES}
+          echo "newrelic/newrelic-agent-init-container - Releasing ${RELEASE_TITLE} with tag ${RELEASE_TAG}"
+          gh release create "${RELEASE_TAG}" --title=${RELEASE_TITLE} --repo=newrelic/newrelic-agent-init-container --notes=${RELEASE_NOTES}
+        env:
+          GH_RELEASE_TOKEN: ${{ secrets.GH_RELEASE_TOKEN }}
```

We don't have a `GH_RELEASE_TOKEN` secret in this repo; this should probably use `DOTNET_AGENT_GH_TOKEN` instead.
```suggestion
          GH_RELEASE_TOKEN: ${{ secrets.DOTNET_AGENT_GH_TOKEN }}
```